### PR TITLE
Update mocha_v2.4.x.js

### PR DESCRIPTION
**Pull request by nodkz**
Mocha: `before` and `after` methods can have description.

### CODE REVIEW

**Comment by jeffmo** [comment]
Can you add a couple of tests that show both forms of these functions working?

**Comment by nodkz** [comment]
Ok try add tests, now.

**Comment by nodkz** [comment]
@jeffmo some crazy error in CI after me. Please check it and give advice.
Can not imagine what I'm done wrong.

Also I am confused with spaces. Can not understand logic should I 
do `description:string` 
or `description : string`.

File: definitions/npm/mocha_v2.4.x/test_mocha_v2.4.x_hooks.js [hunk 1]
```diff
@@ -34,11 +39,14 @@ beforeEach((done : Function) => new Promise((res, rej) => {}));
...
 after((done : Function) => {});
 after(() => new Promise((res, rej) => {}));
+after('some description', () => {});
 
 // $ExpectError number. This type is incompatible with function type.
 after((done : number) => {});
 // $ExpectError can't pass done, and return promise at the same time.
 after((done : Function) => new Promise((res, rej) => {}));
+// $ExpectError number. This type is incompatible with function type.
+after('some description', (done : number) => {});
```
Comment (jeffmo) [inline definitions/npm/mocha_v2.4.x/test_mocha_v2.4.x_hooks.js]: Not sure why you have `$ExpectError` on this line? This line seems valid to me...

(Note that this is also why the tests are failing: You've asserted that you're expecting Flow to give an error on line 49 by putting the `$ExpectError` comment above it)

**Comment by jeffmo** [comment]
> some crazy error in CI after me. Please check it and give advice

I commented inline. It looks like you've asserted expectations of an error in places where Flow isn't actually erroring.

> should I do `description:string` or `description : string`.

I generally do `description: string`

**Comment by marudor** [comment]
This hasn't been updated for quite a while and I can't add commits to the PR.
If this is still relevant please submit a new PR.

**Pull request closed without merging**